spec/memfs/file_spec.rb

```diff
@@ -1803,6 +1803,54 @@ module MemFs
       end
     end
 
+    describe '.write' do
+      it 'writes the string to the given file' do
+        described_class.write('/test-file', "test")
+        read_content = described_class.read('/test-file')
+        expect(read_content).to eq "test"
+      end
+
+      context 'when +offset+ is provided' do
+        it 'writes the string to the given file when offset is 0' do
+          described_class.write('/test-file', "test", 0)
+          read_content = described_class.read('/test-file')
+          expect(read_content).to eq "test"
+        end
+
+        it 'writes the string to the given file when offset is nil' do
+          described_class.write('/test-file', "test", nil)
+          read_content = described_class.read('/test-file')
+          expect(read_content).to eq "test"
+        end
+
+        it 'starts writing from the offset' do
+          pending("Offsets not yet implemented, because Content#write always appends.")
+          described_class.write('/test-file', "test")
+          described_class.write('/test-file', "test", 2)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.